MatrixKit/Controllers/MXKAttachmentsViewController.m

```diff
@@ -686,7 +685,43 @@ - (void)collectionView:(UICollectionView *)collectionView didSelectItemAtIndexPa
+                                                                                    multiplier:1.0f
+                                                                                      constant:0.0f];
+
+                    NSLayoutConstraint *trailingConstraint = [NSLayoutConstraint constraintWithItem:selectedCell.moviePlayer.view
+                                                                                          attribute:NSLayoutAttributeLeading
+                                                                                          relatedBy:0
+                                                                                             toItem:selectedCell.customView
+                                                                                          attribute:NSLayoutAttributeLeading
+                                                                                         multiplier:1.0
+                                                                                           constant:0];
+
+                    NSLayoutConstraint *bottomConstraint = [NSLayoutConstraint constraintWithItem:selectedCell.moviePlayer.view
+                                                                                        attribute:NSLayoutAttributeBottom
+                                                                                        relatedBy:0
+                                                                                           toItem:selectedCell.customView
+                                                                                        attribute:NSLayoutAttributeBottom
+                                                                                       multiplier:1
+                                                                                         constant:0];
+
+                    NSLayoutConstraint *tailingConstraint = [NSLayoutConstraint constraintWithItem:selectedCell.moviePlayer.view
+                                                                                         attribute:NSLayoutAttributeTrailing
+                                                                                         relatedBy:0
+                                                                                            toItem:selectedCell.customView
+                                                                                         attribute:NSLayoutAttributeTrailing
+                                                                                        multiplier:1.0
+                                                                                          constant:0];
                     
+                    selectedCell.moviePlayer.view.translatesAutoresizingMaskIntoConstraints = NO;
+                    [NSLayoutConstraint activateConstraints:@[topConstraint, trailingConstraint, bottomConstraint, tailingConstraint]];
```

Comment:
activateConstraints is not supported on ios7

please refer to deprecated API addConstraint